setup.py

```diff
@@ -13,7 +13,7 @@
     "dask",
     "datacube[s3,performance]",
     "dea_tools>=0.3.6",
-    "eodatasets3<1.9",
```

Comment:
This confuses me a little - isn't it already set like this on `develop` branch? https://github.com/GeoscienceAustralia/dea-intertidal/blob/develop/setup.py#L16

Reply:
Fixed!